noxfile.py

```diff
@@ -286,6 +291,8 @@ def _macos_doctest_install(session, install_prefix):
     # 2. Build the wheel from source.
     basic_dir = tempfile.mkdtemp()
     session.run(
+        "python",
+        "-m",
```

Comment:
I added this when debugging (I was worried something was wrong in `pip` resolution). I didn't end up **needing** this change but `python -m pip` is more canonical so keeping it.